docs/usage/after_provisioning.md

```diff
@@ -329,5 +329,63 @@ If the IPMI static IP addressing was manually configured on the BMCs you have to
 ## IPMI Credentials
 
 The IPMI credentials under `/usr/share/oem/ipmi_(user|password)` are expected to stay valid for all nodes except the management node which doesn't have its BMC in the rack-internal network.
-Since the network is internal, there shouldn't be a need to change weak credentials as long as you disallow untrusted workloads from accessing the internal network.
 If you want to switch to stronger credentials, it's best to create a new IPMI account on all BMCs and when done, update the credential files and only then remove the old account.
+
+First, find out which IPMI slot corresponds to the user name in `/usr/share/oem/ipmi_user`:
+
+```
+$ ipmi --all user list | grep "$(cat /usr/share/oem/ipmi_user)" | uniq
+# The output may be:
+# 2   MYUSER           true    false      true       ADMINISTRATOR
+```
+
+The first number is the slot number which you should not use for the new user because it's safer to keep this user to have a fallback until the new user is fully set up.
+
+Now, find a free slot for the new user:
+
+```
+$ ipmi --all user list | sort -n | uniq
+# The output may be:
+# ID  Name           Callin  Link Auth  IPMI Msg   Channel Priv Limit
+# 1                    true    false      true       USER
+# 2   MYUSER           true    false      true       ADMINISTRATOR
+# 3                    true    false      false      NO ACCESS
+# 4                    true    false      false      NO ACCESS
+# 5                    true    false      false      NO ACCESS
+# 6                    true    false      false      NO ACCESS
+# 7   operator         true    false      false      OPERATOR
+# 8                    true    false      false      NO ACCESS
+# 9                    true    false      false      NO ACCESS
+# 10                   true    false      false      NO ACCESS
+```
+
+The IDs where no name is given are free but you can also repurpose an unused account.
```

Comment:
repurpose an unused account ?

Reply:
If there is an old and unused account, it can be overwritten

Reply:
If the instructions in here can be converted as steps, it would be great. Though no issues if it can't.

Having instructions laid out as steps increase readability.

Reply:
Ok, done